x/ibc/04-channel/keeper/keeper.go

```diff
@@ -62,20 +65,24 @@ func (k Keeper) SetChannel(ctx sdk.Context, portID, channelID string, channel ty
 }
 
 // GetChannelCapability gets a channel's capability key from the store
-func (k Keeper) GetChannelCapability(ctx sdk.Context, portID, channelID string) (string, bool) {
+func (k Keeper) GetChannelCapability(ctx sdk.Context, portID, channelID string) (capability.Capability, bool) {
```

Comment:
This function can be removed, it is superseded by `getCapability`, which can be called on the `ScopedCapabilityKeeper`